data/sql/updates/pending_db_world/Fix-NPCs_Halls_of_Reflection.sql

```diff
@@ -0,0 +1,31 @@
+/*  
+SQL Dark Ranger Loralen emote
+*/  
+UPDATE `creature_template_addon` SET `emote` = 0 WHERE (`entry` = 37779);
+
+/*  
+SQL LK Boss final
+*/  
+UPDATE `creature` SET `guid`=1972025, `id1`=36954, `id2`=0, `id3`=0, `map`=668, `zoneId`=0, `areaId`=0, `spawnMask`=3, `phaseMask`=1, `equipment_id`=1, `position_x`=5553.48, `position_y`=2262.91, `position_z`=733.012, `orientation`=4.03979, `spawntimesecs`=86400, `wander_distance`=0, `currentwaypoint`=0, `curhealth`=27890000, `curmana`=0, `MovementType`=0, `npcflag`=0, `unit_flags`=0, `dynamicflags`=0, `ScriptName`='', `VerifiedBuild`=0 WHERE `guid`=1972025;
+
+/*  
+SQL LK Boss final
+*/
+UPDATE `creature` SET `guid`=1972026, `id1`=37554, `id2`=0, `id3`=0, `map`=668, `zoneId`=0, `areaId`=0, `spawnMask`=1, `phaseMask`=1, `equipment_id`=1, `position_x`=5554.73, `position_y`=2259.63, `position_z`=733.011, `orientation`=1.78626, `spawntimesecs`=300, `wander_distance`=0, `currentwaypoint`=0, `curhealth`=5040000, `curmana`=881400, `MovementType`=0, `npcflag`=0, `unit_flags`=0, `dynamicflags`=0, `ScriptName`='', `VerifiedBuild`=NULL WHERE `guid`=1972026;
```

Comment:
For UPDATES, only change what needs to be changed. position, orientation, spawntime, etc.
Don't change id1 if it's not necessary, and absolutely don't change the guid, since it's the primary key, unless you're absolutely certain of what you're doing.
curhealth and curmana doesn't do anything so they aren't needed here as well.

And don't set VerifiedBuild to NULL, set it to your sniff's build. For example, 48999. It should be located in the parsed txt file.

Reply:
> For UPDATES, only change what needs to be changed. position, orientation, spawntime, etc. Don't change id1 if it's not necessary, and absolutely don't change the guid, since it's the primary key, unless you're absolutely certain of what you're doing. curhealth and curmana doesn't do anything so they aren't needed here as well.
> 
> And don't set VerifiedBuild to NULL, set it to your sniff's build. For example, 48999. It should be located in the parsed txt file.

in the case of the Verified Build, the creatures already didn't have it (I don't know why), but I'm not very good at reading sniffs, I'm still learning. But in the case of this Gossip there, it exists but is not in the database. But I will check all this :)

Reply:
> in the case of the Verified Build, the creatures already didn't have it (I don't know why), but I'm not very good at reading sniffs, I'm still learning. But in the case of this Gossip there, it exists but is not in the database. But I will check all this :)

Even if you were replacing it with values that aren't verified with sniffs, you'd still only have it be a build number or 0, never null.

Reply:
> > in the case of the Verified Build, the creatures already didn't have it (I don't know why), but I'm not very good at reading sniffs, I'm still learning. But in the case of this Gossip there, it exists but is not in the database. But I will check all this :)
> 
> Even if you were replacing it with values that aren't verified with sniffs, you'd still only have it be a build number or 0, never null.

If you want, I can put the SQL created by WowPacketParser here, but I don't know if what I'm reading there is the build number or if it's something else ..... :( 

Anyway, the PR is open to modifications exactly to get these details right and other things, if you can help I'd appreciate it :)

Reply:
At the very top of the parsed .txt file, there will be a line that reads the following:
```
# Detected build: V#_#_#_#
```
For example, in one parse it may look like `V1_14_3_44834`, which would mean patch 1.14.3, build 44834. The VerifiedBuild number will be the last 5-digit number in that row. That's all you need to put in that column.

Reply:
> At the very top of the parsed .txt file, there will be a line that reads the following:
> 
> ```
> # Detected build: V#_#_#_#
> ```
> 
> For example, in one parse it may look like `V1_14_3_44834`, which would mean patch 1.14.3, build 44834. The VerifiedBuild number will be the last 5-digit number in that row. That's all you need to put in that column.

thank you very much!
Updated SQL with builds :) 48999
![image](https://user-images.githubusercontent.com/122452427/235841652-fb8aca19-3975-40af-89a9-160daa084cfe.png)

Reply:
@Gultask @heyitsbench 
everything is ok with the SQL now, I just updated the placement of the creatures as per the Sniff file and added the build as you told me, thanks for the help :)